crypto/obj/objects.txt

```diff
@@ -134,8 +134,11 @@ secg-ellipticCurve 39		: sect571r1
  : ffdhe8192
 
 # PQ Group OIDs from OQS
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/ALGORITHMS.md#oids
```

Comment:
From this page:
> Along the same lines as the code points, X.509 OIDs may be subject to change prior to final standardization

Are there more official values that are not in 1.3.9999 "reserved" section?

Reply:
Not that I'm aware of. There doesn't appear to be official OID's for `X25519MLKEM768` yet, but I believe an OID is needed in order to successfully use these supported groups from Python. I forget the exact reason why, but @WillChilds-Klein should know?

Reply:
the code path is a little convoluted, but if i recall correctly it was:

1. python [calls](https://github.com/WillChilds-Klein/cpython/blob/bd3d31f380cd451a4ab6da5fbfde463fed95b5b5/Modules/_ssl.c#L4386) `OBJ_sn2nid`
2. `OBJ_sn2nid` [calls](https://github.com/aws/aws-lc/blob/9d21f38b8ea04a70a753626708e3de21d8d0b733/crypto/obj/obj.c#L265) `lh_ASN1_OBJET_retrieve(global_added_by_short_name, ...)`
3. that array only gets populated with members that have OIDs

i might be recalling 3. incorrectly, but the functions called by python assumed that all valid "curves" (overloaded term here) had valid OIDs.